lib/Dependencies.js

```diff
@@ -8,14 +8,18 @@ const templates = loadTemplates(path.join(__dirname, '../templates'));
 const authorize = require('./authorize');
 const log = require('./log')();
 const EvidenceStoreGateway = require('./gateways/document/EvidenceStoreGateway');
+const { NotifyClient } = require('notifications-node-client');
 
 const configuration = {
   urlPrefix: process.env.URL_PREFIX,
   evidenceStoreUrl: process.env.EVIDENCE_STORE_URL,
   evidenceStoreToken: process.env.EVIDENCE_STORE_TOKEN,
-  maxUploadBytes: parseInt(process.env.MAX_UPLOAD_BYTES) || 20_971_520
+  maxUploadBytes: parseInt(process.env.MAX_UPLOAD_BYTES) || 20_971_520,
+  gov_notify_template_id: process.env.GOV_NOTIFY_TEMPLATE_ID
```

Comment:
This can be lower camelcase now instead to match the JS convention.
```suggestion
  govNotifyTemplateId: process.env.GOV_NOTIFY_TEMPLATE_ID
```

Reply:
Ah yes! You're right!